src/gtk/widgets/recent_info.rs

```diff
@@ -86,7 +86,7 @@ impl RecentInfo {
         unsafe {
             let mut app_exec = ptr::null();
             let mut count = 0u32;
-            let mut time_ = 0i64;
+            let mut time_: time_t = 0;
```

Comment:
The following patch will fix this:

 - [0003-GtkRecentInfo-types-can-be-inferred.patch](http://www.synkhronix.com/pub/rust/patches/rgtk/0003-GtkRecentInfo-types-can-be-inferred.patch)